README.md

```diff
@@ -286,30 +286,32 @@ The following inductive datasets are built in to PyKEEN.
+| [Negative Likelihood Ratio](https://en.wikipedia.org/wiki/Negative_likelihood_ratio)                                   | [0, ∞)              | 📉          | FNR / TNR                                                   | Classification |
 | [Negative Predictive Value](https://en.wikipedia.org/wiki/Negative_predictive_value)                                   | [0, 1]              | 📈          | TN / (TN + FN)                                              | Classification |
-| [Positive Likelihood Ratio](https://en.wikipedia.org/wiki/Positive_likelihood_ratio)                                   | [0, inf)            | 📈          | TPR / FPR                                                   | Classification |
+| [Positive Likelihood Ratio](https://en.wikipedia.org/wiki/Positive_likelihood_ratio)                                   | [0, ∞)              | 📈          | TPR / FPR                                                   | Classification |
 | [Positive Predictive Value](https://en.wikipedia.org/wiki/Positive_predictive_value)                                   | [0, 1]              | 📈          | TP / (TP + FP)                                              | Classification |
 | [Prevalence Threshold](https://en.wikipedia.org/wiki/Prevalence_threshold)                                             | [0, 1]              | 📉          | √FPR / (√TPR + √FPR)                                        | Classification |
 | [Threat Score](https://rexmex.readthedocs.io/en/latest/modules/root.html#rexmex.metrics.classification.threat_score)   | [0, 1]              | 📈          | TP / (TP + FN + FP)                                         | Classification |
 | [True Negative Rate](https://en.wikipedia.org/wiki/Specificity_(tests))                                                | [0, 1]              | 📈          | TN / (TN + FP)                                              | Classification |
 | [True Positive Rate](https://en.wikipedia.org/wiki/Sensitivity_(test))                                                 | [0, 1]              | 📈          | TP / (TP + FN)                                              | Classification |
 | [Adjusted Arithmetic Mean Rank (AAMR)](https://arxiv.org/abs/2002.06914)                                               | [0, 2)              | 📉          | The mean over all ranks divided by its expected value.      | Ranking        |
 | [Adjusted Arithmetic Mean Rank Index (AAMRI)](https://arxiv.org/abs/2002.06914)                                        | [-1, 1]             | 📈          | The re-indexed adjusted mean rank (AAMR)                    | Ranking        |
+| [Adjusted Geometric Mean Rank Index (AGMRI)](https://arxiv.org/abs/2002.06914)                                         | (-∞, 1]             | 📈          | The re-indexed adjusted geometric mean rank (AGMRI)         | Ranking        |
```

Comment:
shouldn't this also be `(-E[f]/(1-E[f]), 1]`?

Reply:
7b9d4c9